db/migrate/20200603175553_remove_one_of_our_vans_from_incident_reports.rb

```diff
@@ -1,5 +0,0 @@
-class RemoveOneOfOurVansFromIncidentReports < ActiveRecord::Migration[5.2]
-  def change
-    remove_column :incident_reports, :one_of_our_vans, :boolean
-  end
-end
```

Comment:
Why are you deleting two migrations here? Deleting the addition and removal of a column may not hurt, but there's certainly a chance it will and more certainly no reason to do so.